Resources/Storage/test/Test_StorageElement.py

```diff
@@ -81,6 +81,79 @@ def mock_StorageFactory_generateStorageObject( storageName, pluginName, paramete
 
   return S_OK( storageObj )
 
+def mock_StorageFactory_getConfigStorageName( storageName, referenceType, rootConfigPath ):
+  return S_OK( storageName )
+
+def mock_StorageFactory_getConfigStorageOptions( storageName, rootConfigPath, derivedStorageName = None ):
+  """ Get the options associated to the StorageElement as defined in the CS
+  """
+
+  options = {'BackendType': 'local',
+                            'ReadAccess': 'Active',
+                            'WriteAccess': 'Active',
+            }
+
+  if storageName in ( 'StorageE', ):
+    options['WriteProtocols'] = ['root', 'srm']
+
+
+
+
+  return S_OK( options )
+
+def mock_StorageFactory_getConfigStorageProtocols( storageName, rootConfigPath, derivedStorageName = None ):
```

Comment:
Alexandre will redo the PR anyway.
But concerning that, I think we once need to really have discussion on the topic, but I highly doubt that one can have mock that works for several use cases. To be seen